packer/HWIMO-BUILD

```diff
@@ -187,7 +189,14 @@ fi
 
 
 # execute 'packer build'
-echo ${PACKER_TEMP} | $PACKER build --force --only=${BUILD_TYPE}-iso --var-file=$CFG_FILE  - | $TS | tee packer-install.log 
+if [ "$BUILD_TYPE"  != "vmware" ]
+then
+    echo ${PACKER_TEMP} | $PACKER build --force --only=${BUILD_TYPE}-${FROM_SRC} --var-file=$CFG_FILE  - | $TS | tee packer-install.log 
+else
+    echo ${PACKER_TEMP} | $PACKER build --force --only=${BUILD_TYPE}-iso  --var-file=$CFG_FILE  - | $TS | tee packer-install.log
```

Comment:
👎 
We should make the packer script configurable for 1-stage build and 2-stage build.
say, the external community may need 1-stage build, because they don't have licensed VMWare Workstation, which not support 2-stage build.